macros/Glossary.ejs

```diff
@@ -8,12 +8,12 @@
 //  $1 - name to display (optional)
 //  $2 - Remove custom style on links; boolean (optional)
 
-var str = $1 || $0;
-var css = !$2 ? "glossaryLink" : "";
+let str = $1 || $0;
+let css = !$2 ? "glossaryLink" : "";
 
 // Terme location
-var URL_EN = "/en-US/docs/Glossary/";
-var URL    = mdn.localString({
+let URL_EN = "/en-US/docs/Glossary/";
```

Comment:
@davidflanagan Should we define these as `const` perhaps?

Reply:
Probably.

Reply:
I'd rather not make any more macro changes in this PR, especially since it risks changing the behavior of the macros in subtle ways.